sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -62,12 +62,17 @@ export class BatchingReceiver extends MessageReceiver {
   /**
    * Receives a batch of messages from a ServiceBus Queue/Topic.
    * @param maxMessageCount      The maximum number of messages to receive.
+   * This has to be a number <= 2047 for receiver in `peekLock` mode, due to memory size
+   * constraints of underlying buffer.
+   * If a value > 2047 is used, then only 2047 messages are fetched at a time.
```

Comment:
Suggestion: 
> number <= 2047

`number less than or equal to 2047`

> If a value > 2047 is used

`If a value greater than 2047 is used`

Reply:
Updated

Reply:
I am not seeing the updated text here.

Reply:
Looks like commit didn't get pushed, should be updated now